flipperdevices/flipperzero-firmware
# [FL-1293] iButton key rename

# What's new

- In Key Edit menu delete existing file before saving new
- Save key name in iButtonKey object
- Clear key name and data

# Verification 

- Build for f5 and f6 targets
- Edit saved key. Verify correct key list in Saved menu
- Add manually key after reading and naming the key. Verify that pre-set key data is zero and name is random

# Checklist (do not modify)
